common-queue/src/main/java/org/duracloud/common/queue/aws/SQSTaskQueue.java

```diff
@@ -79,10 +80,10 @@
      * http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/sqs/AmazonSQSClient.html#AmazonSQSClient()
```

Comment:
Looks like this link should be updated to: http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html. 

It might also be useful to mention here that the region can be set via the AWS_REGION environment variable or one of the other methods described here: http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-region-selection.html